apps/parsley/src/types/logs.ts

```diff
@@ -3,7 +3,24 @@ import { CaseSensitivity, MatchType } from "constants/enums";
 type ExpandedLine = [number, number];
 type ExpandedLines = ExpandedLine[];
 
-type ProcessedLogLine = number | number[];
+/**
+ * Represents a range of lines in a log file.
+ * {Object} Range
+ * {number} lineStart - The starting line inclusive of the range.
+ * {number} lineEnd - The ending line exclusive of the range.
+ */
+interface Range {
+  lineStart: number;
+  lineEnd: number;
```

Comment:
nit. For js doc comments to work on interfaces you must apply the comments directly to the field.
```suggestion
interface Range {
  /** The starting line inclusive of the range */
  lineStart: number;
  /** The ending line exclusive of the range */
  lineEnd: number;
```

<img width="702" alt="image" src="https://github.com/evergreen-ci/ui/assets/4605522/1723cc88-dc4d-4a81-894b-90494cc8729e">